src/pages/PathSelection.test.js

```diff
@@ -129,4 +135,25 @@ describe('PathSelection Component UI Tests', () => {
       ).toBeInTheDocument();
     });
   });
+
+  it('should retrieve all paths from the database', async () => {
```

Comment:
En edes ymmärrä miten tämä testi menee läpi. Eihän tässä testissä aseteta tietokantaan mitään polkuja? 
Muutenkin parempi testin nimi tälle skenaariolle on esim "should navigate to edit path page" tai vastaava.

Reply:
Oho, tohon jäi väärä otsikko ku jatkokehitin sitä. Eli ihan vaa huolimattomuusvirhe.